src/App.tsx

```diff
@@ -49,7 +49,7 @@ const App: FC = () => {
       load(RUDDERSTACK_WRITE_KEY, RUDDERSTACK_DATA_PLANE_URL)
     }
     setAutoFreeze(false)
-  }, [RUDDERSTACK_WRITE_KEY, RUDDERSTACK_DATA_PLANE_URL])
+  }, [])
```

Comment:
these were unnecessary in the useEffect since they should never change (linter was throwing a warning)